components/campaigns/AnnoucementHero.js

```diff
@@ -29,22 +29,24 @@ export default function AnnouncementHero({ className = '', small = false, hideVi
         className={`bg-gray-50 border border-gray-200 py-6 rounded ${className} ${
           small ? 'mb-4' : 'mb-12'
         }`}
+        data-testid = "container-div"
       >
         <Heading
           className="countdown-text-gradient"
           level="h2"
           typeStyle="heading-lg"
+        
         >
           AsyncAPI Conf on Tour 2023
         </Heading>
         <Heading
           className="countdown-text-gradient"
           level="h3"
           typeStyle="heading-sm"
-        >
+          data-testid="sub-heading "     >
```

Comment:
remove the extra trailing space @reachaadrika

Reply:
resolved done